package.json

```diff
@@ -3,7 +3,9 @@
   "version": "0.0.1",
   "description": "A starting point for the CSS Frameworks CA project",
```

Comment:
Please update your package description to describe what you have created

Reply:
started all over again: description updated